build/websocket-scenarios.yml

```
@@ -16,8 +16,14 @@ parameters:
   type: object
   default: 
 
-  - displayName: Echo
-    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --property scenario=echo
+  - displayName: Echo-10000
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=10000--property scenario=echo
```

```suggestion
    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=10000 --property scenario=echo --property size=1000 --property compress=false
```

Space, but also scenario is a metadata in the results, used by power bi only. I assume you want to expose "size" as a different list, but we could also change the "scenario" metadata to contain the size (echo-1000). You choose (not separate is always better as we can still concatenate in PBI)